packages/manager/apps/pci-rancher/public/translations/dashboard/Messages_fr_FR.json

```diff
@@ -41,5 +41,11 @@
   "createRancherError": "Une erreur est survenue lors de la création de votre Managed Rancher Service. {{rancherCreationErrorMessage}}.",
   "createRancherRecomendedVersion": "Version Recommandée",
   "createRancherOVHCloudPlanDescription": "Créez ou gérez des « downstream clusters » Managed Kubernetes Service ou « self managed » sur le cloud public OVHcloud ou vos propres machines. Vous pouvez aussi enregistrer des clusters externes (sans gestion cloud)",
-  "createRancherStandardPlanDescription": "Créez ou enregistrez des « downstream clusters » avec n’importe quel cloud public supporté par Rancher ou sur un cloud privé automatisable (ex. VMware vSphere), y compris les scénarios supportés par l’offre OVHcloud Edition."
+  "createRancherStandardPlanDescription": "Créez ou enregistrez des « downstream clusters » avec n’importe quel cloud public supporté par Rancher ou sur un cloud privé automatisable (ex. VMware vSphere), y compris les scénarios supportés par l’offre OVHcloud Edition.",
+  "updateOfferModalCancelButton": "Annuler",
+  "updateOfferModalValidateButton": "Confirmer",
+  "updateOfferModaleTitle": "Changer d'offre",
+  "updateOfferModalDescription": "Vous êtes sur le point de modifier votre offre actuelle pour l'offre {{availableServicePlan}}. Vous serez facturé {{planPricing}} par vCPUs orchestrés. Cette opération prendra effet immédiatement et n'a pas d'impact sur vos « downstream clusters » ni les données de votre Rancher managé. Veuillez confirmer si vous souhaitez continuer.",
```

Comment:
```suggestion
  "updateOfferModalDescription": "Vous êtes sur le point de modifier votre offre actuelle pour l'offre {{availableServicePlan}}. Vous serez facturé {{planPricing}} par vCPU orchestré. Cette opération prendra effet immédiatement et n'a pas d'impact sur vos « downstream clusters » ni les données de votre Rancher managé. Veuillez confirmer si vous souhaitez continuer.",
```

Reply:
That was the text written by the PO in the lira